botorch_community/models/vbll_helper.py

```diff
@@ -5,531 +5,475 @@
 # LICENSE file in the root directory of this source tree.
 
 """
-The following code is from the repository vbll (https://github.com/VectorInstitute/vbll)
-which is under the MIT license.
+Variational Bayesian Last Layers — enhanced port.
+
+Original: vbll (https://github.com/VectorInstitute/vbll), MIT license.
 Paper: "Variational Bayesian Last Layers" by Harrison et al., ICLR 2024
+
+Enhancements:
```

Comment:
Are these enhancements between the new version and the vbll original, or between the new version and the old BoTorch version? Or are the vbll original and the BoTorch version the same?